POEM_068.md

```diff
@@ -0,0 +1,198 @@
+POEM ID: 068  
+Title:  Nonlinear Solver System Output Caching  
+authors: @lamkina  
+Competing POEMs:  
+Related POEMs:  
+Associated implementation PR: (not submitted yet)  
+
+Status:
+
+- [x] Active
+- [ ] Requesting decision
+- [ ] Accepted
+- [ ] Rejected
+- [ ] Integrated
+
+
+
+## Motivation
+
+Many optimizers employ backtracking algorithms to recover from failed model evaluations.
+After a failed evaluation, the user in conjunction with OpenMDAO must decide how to initialize the model's states
+for the next backtracking iteration.  Three main options exist:
+
+1. **Cold Start:**  Use the initial guesses to the model at the onset of the optimization problem.
```

Comment:
I just had a quick chat with @lamkina about this. A few things that I brought up:
- In the optimization world, personally I consider "warm start" a different approach to what's described here. Cold vs. warm vs. hot refers to the _quantity_ of information supplied to initialize some model, not the _quality_, so the warm start described here would actually be a cold start with a different set of states. That being said, I'm not sure if the same translates to OM and there's little reason to change established terminology.
- I would prefer a different name to "restart" though, since it's not very descriptive and warm start can be described as a restart strategy also. @lamkina suggested something along the lines of the following:
  - `warm start` -> `restart from previous`
  - `restart` -> `restart from successful`
- whatever is picked here should match the name of the actual option (currently `use_cached_states`).

Reply:
The terminology should be changed to maintain consistency with @nwu63 definition.  We should be describing the quantity or type of inormation being supplied as well as from where the information originates.